docs/pages/worked-examples/intro_set.md

```diff
@@ -0,0 +1,168 @@
+"contactName": "Chris Bielow",
+"contactAddress": "chris.bielow@bsc.fu-berlin.de",
+"description": "A simple mzQC file containing information for a set of multiple mass spectrometry runs.",
+```
+
+2. **Controlled vocabulary (CV) references:** This section points to standardized vocabularies used to ensure consistent metric definitions across files.
+It is typically included at the end of the mzQC file.
+
+```
+"controlledVocabularies": [
+  {
+    "name": "Proteomics Standards Initiative Mass Spectrometry Ontology",
+    "uri": "https://github.com/HUPO-PSI/psi-ms-CV/releases/download/v4.1.130/psi-ms.obo",
+    "version": "4.1.130"
+  }
+]
+```
+
+3. **Quality metrics for the set:** This core part of the file captures the QC metrics specific to the set being described.
+
+```
+"setQualities": [
+  {
+    ...
+  }
+]
+```
+
+Each element within `setQualities` defines a distinct set, enabling the comparison of, say, different experimental conditions or replicate groups.
+
+A set's QC data is contextual—it makes sense within the bounds of the group.
+For instance, it wouldn't be right to lump individual run metrics like MS1 scan counts for several runs into a single set metric; those belong to individual run analyses.
+
+Imagine you have several technical replicates from an experiment with two conditions, and you're interested in grouping these by technical replicates.
+You might end up with sets for "healthy" and "diseased" conditions, plus a combined "all" set for overarching analyses.
+As an example, we'll use three different groupings:
+1. The "healthy" set, consisting of technical replicates "techRep1_healthy", "techRep2_healthy", "techRep3_healthy".
+2. The "diseased" set, consisting of technical replicates "techRep1_diseased", "techRep2_diseased", "techRep3_diseased".
+3. The "all" set, combining both the "healthy" and "diseased" set.
+
+These labels are important, acting as descriptive tags for each set, guiding your analysis.
+Therefore, it is recommended to use a descriptive label, for example based on the experimental design or the kind of comparisons you want to make.
+
+```
+"metadata": {
+  "label": "healthy",
+  "inputFiles": [
+    ...
+  ]
+},
+"qualityMetrics": [
+  ...
+]
+```
+
+`inputFiles` lists the specific files contributing to a set, with all the technical details neatly described using CV terms.
+
+```
+"inputFiles": [
+  {
+    "name": "techRep1_healthy",
+    "location": "file://C:/msdata/techRep1_healthy.mzML",
+    ...
+  },
+  {
+    "name": "techRep2_healthy",
+    "location": "file://C:/msdata/techRep2_healthy.mzML",
+    ...
+  },
+  {
+    "name": "techRep3_healthy",
+    "location": "file://C:/msdata/techRep3_healthy.mzML",
+    ...
+  }
+],
+```
+
+Let's dive into an example metric, like the "protein contaminant intensity ratio," indicating how much of your sample is taken up by known contaminants. A higher value suggests more contamination:
+
+```
+{
+  "accession": "MS:4000XXX",
```

Comment:
lets wait for https://github.com/HUPO-PSI/psi-ms-CV/pull/246 to get a definitive number here